woodwork/table_accessor.py

```diff
@@ -600,7 +600,7 @@ def wrapper(*args, **kwargs):
                                       TypingInfoMismatchWarning)
                     else:
                         copied_schema = self._schema._get_subset_schema(list(self._dataframe.columns))
```

Comment:
yes, but I think the idea is that we wanted to be explicit about making a copy here (though maybe the variable name does a good enough job of that)

Reply:
No strong preference on my end, but I'm leaning toward `copied_schema = self.schema` as a slightly better option. As you say the variable name provides a clue that this is a copy, and the schema property code and docstring clearly say it returns a copy. That update would also mean we aren't using `_get_subset_schema` to get a full schema instead of a subset.

Reply:
ok switched to using `self.schema`!